core/scss/settings/settings.scss

```diff
@@ -4244,10 +4241,10 @@ $definition-list-margin-bottom: 0.5rem;
 
 // ex.: golden ratio
 // current: 16 * 1.71429 / 4 = 6.85716
-$base-scale: $base-unit-rem * $base-line-height-ratio / $base-unit-px;
+$base-scale: $base-unit-rem * calc($base-line-height-ratio / $base-unit-px);
```

Comment:
Tests are failing on these...I think you have to use `#{$base-line-height-ratio}` format when referencing variables in `calc()`

Reply:
got it, thank you for the catch